src/vnet_proxy.erl

```diff
@@ -31,13 +32,11 @@ start(VNodeC, VNodeS, Pid, Gen) ->
 
 -spec on_disconnect(pid()) -> ok.
 on_disconnect(Proxy) ->
-  Proxy ! ?noconnection(true),
-  ok.
+  sync_noconnection(Proxy, true).
 
 -spec on_stop(pid()) -> ok.
 on_stop(Proxy) ->
-  Proxy ! ?noconnection(false),
-  ok.
+  sync_noconnection(Proxy, false).
```

Comment:
When I am reading this, `false` (and `true` right above) hide the meaning of the parameter. Why not use `disconnect` and `stop` instead?